controllers/controllers.js

```diff
@@ -61,4 +61,14 @@ exports.patchArticle = (req, res, next) => {
     selectUsers().then(users => {
       res.status(200).send({users})
     });
-  };
\ No newline at end of file
+
+  };
+
+  exports.deleteComment = (req, res,next) => {
+    const comment_id = req.params.comment_id
+    removeComment(comment_id).then(comment => {
+        res.status(204).send()
```

Comment:
we can delete the .send() here, the status 204 is proof enough of a successful delete